ethereum/core/src/main/java/tech/pegasys/teku/core/epoch/EpochProcessorUtil.java

```diff
@@ -304,6 +305,11 @@ public static void process_registry_updates(MutableBeaconState state)
   public static void process_slashings(MutableBeaconState state) {
     UInt64 epoch = get_current_epoch(state);
     UInt64 total_balance = get_total_active_balance(state);
+    UInt64 adjusted_total_slashing_balance =
+        UInt64.valueOf(
+                state.getSlashings().stream().mapToLong(UInt64::longValue).sum()
+                    * Constants.PROPORTIONAL_SLASHING_MULTIPLIER)
+            .min(total_balance);
```

Comment:
We should be able to avoid manipulating raw `long` values here:
```suggestion
    UInt64 adjusted_total_slashing_balance = state.getSlashings().stream()
            .reduce(UInt64.ZERO, UInt64::plus)
            .times(Constants.PROPORTIONAL_SLASHING_MULTIPLIER)
            .min(total_balance);
```